metrics/bigquery.py

```diff
@@ -82,7 +82,7 @@ def do_query(self, query, out_filename):
         cmd = [
             'bq', 'query', '--format=prettyjson',
             '--project_id=%s' % self.project,
-            '-n100000',  # Results may have more than 100 rows
+            '-n1000000',  # Results may have more than 100 rows
```

Comment:
what does this actually mean?

Reply:
suggestion: `--max_rows=1000000`, it's the same thing, but clearer without cross referencing the BQ docs at https://cloud.google.com/bigquery/docs/reference/bq-cli-reference

Reply:
max number of rows to get back, I bumped the limit up by an order of magnitude without looking more deeply when my two-year backfill came back with incomplete data